src/main/java/com/tamaized/voidfog/FogParticleSpawner.java

```diff
@@ -22,7 +22,7 @@ public void update(World world, Entity entity, Voidable dimension) {
 
         Random rand = world.getRandom();
 
-        for (int pass = 0; pass < VoidFog.config.voidParticleDensity; pass++) {
+        for (int pass = 0; pass < VoidFog.config.voidParticleDensity; pass++) {            
```

Comment:
Revert this change please

Reply:
Reverted in [8d465f2](https://github.com/Sollace/Void-Fog/pull/13/commits/8d465f21f9257c1788b6d29917d657585876508b)